tests/fsharpqa/Source/ErrorMessages/NameResolution/E_GlobalQualifierAfterDot.fs

```diff
@@ -3,4 +3,4 @@
 
 let x = global.System.String.Empty.global.System.String.Empty
 
-exit 0
```

Comment:
@kmutagene,

Do you propose following up with a PR to migrate this test case?  Feel free to add whatever is necessary to the CompilerAssert class.

Thanks

Kevin

Reply:
the new function is already coming in https://github.com/dotnet/fsharp/pull/7244 - so after merging that one @kMutagene can continue

Reply:
OK the new assertion function is in.

Reply:
thanks @forki